src/components/buttons/cancel-button.ts

```diff
@@ -1,4 +1,4 @@
-import { CANCEL_BUTTON } from "@constants";
+import { CLOSE_BTN_IMG } from '@constants';
```

Comment:
same comment. sorry i didn't see earlier.

Reply:
Changes applied, thank you.